openshift/apicast-build-template.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: apicast-build
+metadata:
+  annotations:
+    description: |-
+      A BuildConfig for OpenShift that builds images of Apicast from source.
```

Comment:
@maneta do we want to do something with those annotations? Adding some metadata, tags, ...

Reply:
yes, the guidelines I have from product are:

```yaml
openshift.io/display-name: 3scale APIcast API Gateway
openshift.io/provider-display-name: Red Hat, Inc.
iconClass: icon-3scale
description: >-
  3scale's APIcast is an NGINX based API gateway used to integrate your internal and external
  API services with 3scale's API Management Platform. It supports OpenID connect to integrate
  with external Identity Providers such as Red Hat Single Sign On, for API traffic authentication.
```